framework/src/transfers/MultiAppFieldTransferInterface.C

```diff
@@ -0,0 +1,102 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// MOOSE includes
+#include "MultiAppFieldTransferInterface.h"
+#include "MooseTypes.h"
+#include "FEProblem.h"
+#include "MultiApp.h"
+#include "MooseMesh.h"
+
+template <>
+InputParameters
+validParams<MultiAppFieldTransferInterface>()
+{
+  InputParameters params = validParams<MultiAppTransfer>();
+  params.addRequiredParam<AuxVariableName>(
+      "variable", "The auxiliary variable to store the transferred values in.");
+  params.addRequiredParam<VariableName>("source_variable", "The variable to transfer from.");
```

Comment:
Is it possible to make this parameter and the above as vector of variable names?

Reply:
Definitely, it is possible. But from my understanding, most current field transfers such as the interpolation transfer and the nearestnode  transfer take one nonlinear variable and transfer it to an auxiliary. My implementation simply follow this style.  

If you want to transfer  multiple variables, and you could just simply create multiple transfers in  your input file.